src/conda_rpms/conda_compat.py

```diff
@@ -76,10 +78,20 @@ def read_conda_lockfile(lockfile: Iterable[str]) -> List[str]:
     return urls
 
 
+def read_conda_environment_yml(env_yml: Iterable[str]) -> CondaEnvironment:
+    """Returns a CondaEnvironemnt instance using metadata from the 
+    environment.yml file.
+
+    """
+    env = yaml.safe_load(env_yml)
+    return CondaEnvironment(*env['name'].split('-'), env['meta']['released'],
```

Comment:
Note `env['meta']['released']` is now being used as the tag. That may be a problem if the tag is something like `2022_01_01-1`?

Reply:
This would be the case if we wanted to release two versions of the same environment on the same day? Seems unlikely.